src/_base/harness/attributes/common.yml

```
@@ -156,6 +156,22 @@ attributes.default:
   lighthouse:
     target:
       url: = @('hostname')
+    success-thresholds:
+      pwa:
+        enabled: 0
+        score: 0
+      seo:
+        enabled: 0
+        score: 0
+      best-practices:
+        enabled: 0
+        score: 0
+      accessibility:
+        enabled: 0
+        score: 0
+      performance:
+        enabled: 0
+        score: 0
```

For consistency with other true/false values:
```suggestion
    success-thresholds:
      pwa:
        enabled: false
        score: 0
      seo:
        enabled: false
        score: 0
      best-practices:
        enabled: false
        score: 0
      accessibility:
        enabled: false
        score: 0
      performance:
        enabled: false
        score: 0
```